cellprofiler/object.py

```diff
@@ -40,6 +40,10 @@ def __init__(self):
         self.__parent_image = None
 
     @property
+    def masked(self):
+        return self.segmented & self.parent_image.mask
```

Comment:
Using the `&` operation I get:

```python
data = numpy.ones((5, 5))
mask = numpy.zeros((5, 5), dtype='bool')
data & mask

TypeError: ufunc 'bitwise_and' not supported for the input types, and the inputs could not be safely coerced to any supported types according to the casting rule ''safe''
```

Perhaps `numpy.logical_and` is a better option.